mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -1,10 +1,10 @@
 CLASS net/minecraft/class_238 net/minecraft/util/math/Box
-	FIELD field_1320 maxX D
-	FIELD field_1321 minZ D
-	FIELD field_1322 minY D
-	FIELD field_1323 minX D
-	FIELD field_1324 maxZ D
-	FIELD field_1325 maxY D
+	FIELD field_1320 x2 D
+	FIELD field_1321 z1 D
+	FIELD field_1322 y1 D
+	FIELD field_1323 x1 D
+	FIELD field_1324 z2 D
+	FIELD field_1325 y2 D
```

Comment:
why 1 and 2 instead of 0 and 1?

Reply:
Why 0 and 1 rather than 1 and 2? There doesn't seem to be a consensus on how these should be named, there are currently many things named `x2`. I opened an issue here: https://github.com/FabricMC/yarn/issues/941